eng/Directory.Build.Common.props

```diff
@@ -50,6 +50,13 @@
       CA2000; <!-- Call dispose on IDisposable objects -->
       CA2012; <!-- ValueTask should only be awaited once - conflicts with EnsureCompleted check -->
     </NoWarn>
+    <!--
+      
```

Comment:
Looks like you were going to comment as to "why".

Reply:
Yes I got interrupted while making the edit on github.